### ISSUE TITLE
Error when running example   'magrittr_pipe' not found

### ISSUE THREAD

**Issue by AdamElderfield** [yonicd/ggedit#28]
Hi ,

when I run the example below, I get the following error - any ideas how to solve it? I don't have this problem using the pipe in any other package?

Thanks 

Adam

```
 library('ggedit')
 library(ggplot2)
 p <- ggplot(mtcars, aes(x = hp, y = wt)) + geom_point() + geom_smooth()
 p2 <- ggedit(p)
```

Listening on http://127.0.0.1:3926
Warning: Error in %>%: object 'magrittr_pipe' not found
  56: %>%
  55: cloneLayer
  54: FUN
  53: lapply
  52: FUN
  51: lapply
  50: server
Error in proto_features(l) %>% dplyr::left_join(geom_opts %>% dplyr::filter(!grepl("^stat",  : 
  object 'magrittr_pipe' not found

**Comment by yonicd** [comment]
can you add `sessionInfo()` please

**Comment by yonicd** [comment]
package is currently being updated for the upcoming release of ggplot2. This issue will resolve itself once the new versions come out.

**Comment by yonicd** [comment]
resolved #30
